Marlin/src/lcd/e3v2/proui/dwin.cpp

```diff
@@ -3478,12 +3483,15 @@ void drawMotionMenu() {
 
   void drawManualMeshMenu() {
     checkkey = ID_Menu;
-    if (SET_MENU(manualMesh, MSG_UBL_MANUAL_MESH, 6)) {
+    if (SET_MENU(manualMesh, MSG_UBL_MANUAL_MESH, 7)) {
       BACK_ITEM(drawPrepareMenu);
       MENU_ITEM(ICON_ManualMesh, MSG_LEVEL_BED, onDrawMenuItem, manualMeshStart);
       mMeshMoveZItem = EDIT_ITEM(ICON_Zoffset, MSG_MOVE_Z, onDrawMMeshMoveZ, setMMeshMoveZ, &current_position.z);
       MENU_ITEM(ICON_Axis, MSG_UBL_CONTINUE_MESH, onDrawMenuItem, manualMeshContinue);
       MENU_ITEM(ICON_MeshViewer, MSG_MESH_VIEW, onDrawSubMenu, dwinMeshViewer);
+      #if USE_GRID_MESHVIEWER
+        EDIT_ITEM(ICON_PrintSize, MSG_CHANGE_MESH_VIEWER, onDrawChkbMenu, setViewMesh, &bedLevelTools.view_mesh);
```

Comment:
In what way does this "change" the mesh viewer?

Reply:
there are two types of ways to view the mesh

<p><a target="_blank" rel="noopener noreferrer nofollow"><img align="left" src="https://firebasestorage.googleapis.com/v0/b/vsix-e2247.appspot.com/o/20230601_024101.jpg?alt=media&token=e3e4d698-c94e-45e7-a737-40cc7cee8a9c&_gl=1*avq7fv*_ga*MTE1MDg0NDQ3My4xNjg1ODIwMDE3*_ga_CW55HF8NVT*MTY4NTgyMDAxNi4xLjEuMTY4NTgyMTQyMC4wLjAuMA.." width="270" height="480" style="max-width: 100%;"></a></p>
<p><a target="_blank" rel="noopener noreferrer nofollow"><img src="https://firebasestorage.googleapis.com/v0/b/vsix-e2247.appspot.com/o/20230601_024312.jpg?alt=media&token=cbf0e249-625c-4b53-ba59-3c3d9baae963&_gl=1*1u5bnq0*_ga*MTE1MDg0NDQ3My4xNjg1ODIwMDE3*_ga_CW55HF8NVT*MTY4NTgyMDAxNi4xLjEuMTY4NTgyMTU2OS4wLjAuMA.." width="270" height="480" style="max-width: 100%;"></a><p>
This shows the two types of meshviewer as an example<br>

Reply:
The display with solid colors is much nicer. The display with circles seems noisy and inferior, but is it useful for people with color blindness?

Reply:
Anyway, you get my point. The label "Change Mesh Viewer" with a checkbox next to it is unclear and therefore bad UI design. The label should indicate what type of mesh viewer will be seen when the checkbox is checked. For example, the label "Ugly Mesh View" would make it clear that when the checkbox is checked the mesh viewer will be the "ugly" version, as opposed to the "normal" version.

Reply:
> The display with solid colors is much nicer. The display with circles seems noisy and inferior, but is it useful for people with color blindness?

That's a valid point. I'm not partial to either, just offering making the options more available. I just wanted to say the picture examples posted aren't reminiscent of a true bed level, merely did a quick manual edit of the values to give a quick example of how it can be displayed.

<p><a target="_blank" rel="noopener noreferrer nofollow"><img align="left" src="https://github.com/MarlinFirmware/Marlin/assets/18502096/2b0def19-83ee-4620-acf4-af60613ef7cd)" width="270" height="480" style="max-width: 100%;"></a></p>

this above image is more a true representation of a typical mesh.

anyway it is what it is. and as for calling it "Normal Mesh View" or something like that is a good idea. Having the checkbox enabled means to have the Square block style. default would be the box unchecked and Circle colorful style.